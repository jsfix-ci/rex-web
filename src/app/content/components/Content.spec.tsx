import React from 'react';
import renderer from 'react-test-renderer';
import Content from './Content';

describe('content', () => {
  it('matches snapshot', () => {
    const tree = renderer
      .create(<Content />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
