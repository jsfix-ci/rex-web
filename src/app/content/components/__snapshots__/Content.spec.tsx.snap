// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`content matches snapshot 1`] = `null`;
